Improve instructions for running eVaka locally (#49)
espoon-voltti/evaka: compose/README.md

environment's package manager, or alternatively download binaries
from websites provided below.

Please note that all the dependencies should be installed as 
current user. Do not use elevated privileges, e.g. sudo to install
packages.

- [Node.js](https://nodejs.org/en/) – a JavaScript runtime built on Chrome's V8 JavaScript engine, version  12.13+
- [Yarn](https://yarnpkg.com/getting-started/install) – Package manager for Node, version 1.22.10+
- [JDK](https://openjdk.java.net/projects/jdk/11/) – Java Development
  Kit, version 11+. We recommend using OpenJDK implementation of JSR 384.
- [Docker](https://docs.docker.com/get-docker/) – Docker is an open platform for developing, shipping, and running applications.

### Required tooling

eVaka uses [PM2](https://pm2.keymetrics.io/) for running multiple
services in parallel. PM2 is an opinionated choice for
development process in eVaka. Installing PM2 is not strictly

> mandatory. You can also run every sub-project in separate processes or

Reviewer: ```suggestion
mandatory. You can also run every sub-project separately or
```
They're always technically in separate processes.

> build Docker images and run the stack using Docker (please refer to the
> [documentation for this](#running-the-full-stack-for-e2e-tests)).

Reviewer: Maybe just simplify:

```suggestion
build Docker images and [run the stack using Docker](#running-the-full-stack-for-e2e-tests)).
```

Install PM2 by running following command

```bash
npm install -g pm2
```

> You will also need `nc/netcat` (Arbitrary TCP and UDP connections and listens) utility.

Reviewer: ```suggestion
You will also need the `nc/netcat` (Arbitrary TCP and UDP connections and listens) utility.
```

## Troubleshooting

### Docker

If you are having troubles starting Docker, make sure the user is added
to `docker` group. See [post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/) 
for further reference.

> 

Reviewer: Is this really necessary to document here? This is pretty generic Docker installation stuff

Author: I guess not 😄
